test/synthetics/latest_release_cdn.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * API Synthetic check based on https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/scripting-monitors/write-synthetic-api-tests/
+ * Flow:
+ *   1. Get latest release tag from Github for infra-agent and integrations.
+ *   2. Validate the right package is published in download server.
+ * 
+ * How to install:
+ *   1. Go to Synthetics > Create monitor in NR1.
+ *   2. Select Scripted API check.
+ *   3. Define appropiate settings and copy/paste this script as last step.
+ *   4. Optionally, create an Alert Condition to get notifications.
+ */
+var assert = require('assert');
+var downloadServerBaseURI = `https://download.newrelic.com/infrastructure_agent`;
+var githubOrg = `https://api.github.com/repos/newrelic`;
+var allRepos = ['infrastructure-agent','nri-jmx', 'nrjmx', 'nri-snmp'];
+/* 
+ * Other OHIs: to consider in a separate synthetic check and alert
+'nri-mysql', 'nri-apache', 'nri-cassandra', 'nri-consul', 'nri-couchbase', 
+'nri-elasticsearch', 'nri-f5', 'nri-haproxy', 'nri-kafka', 'nri-memcached', 
+'nri-mongodb', 'nri-mssql', 'nri-mysql', 'nri-nagios', 'nri-nginx', 'nri-oracledb', 
+'nri-postgresql', 'nri-rabbitmq', 'nri-redis'];
+*/
+var arm64Unsupported = ['nri-oracledb'];
+var windowsUnsupported = ['nri-oracledb'];
+
+function checkLatestReleases(repos){
+  repos.forEach(repo => {
+      console.log(`Getting ${githubOrg}/${repo}/releases/latest`);
+      $http.get({
+        'uri': `${githubOrg}/${repo}/releases/latest`,
+        'headers': {'User-Agent': 'newrelic-ohai'}
+      },
+        function (err, response, body) {
+          assert.equal(response.statusCode, 200, `Expected 200 getting releases got ${response.statusCode} ${err}`);
+          
+          // Our release tags might or not contain a 'v' prefix (1.0.0 vs v1.0.0)
+          var latestRelease = JSON.parse(body).tag_name.replace('v','');
+          checkPackagesInCDN(repo, latestRelease);
+      }
+    );
+  });
+}
+
+function checkPackagesInCDN(repo, latestRelease){
+    // need to consider arm64 vs aarch64 vs noarch alternatives
+    var archs = [
+     {
+       archInUrl: 'x86_64',
+       archInPackage: repo === 'nrjmx' ? 'noarch' : 'x86_64'
+    }];
+    if (arm64Unsupported.indexOf(repo) < 0){
+      archs.push({ 
```

Comment:
Why are we doing this thing twice? Should it be arm64? (and arm32)

Reply:
this is wrong, I'll fix it. The 2nd item should be for arm which for some repos we have different convention in URL (aarch64) vs package (arm64), in example SLES: http://download.newrelic.com/infrastructure_agent/linux/zypp/sles/12.4/aarch64/newrelic-infra-1.20.7-1.sles12.4.arm64.rpm